.github/workflows/run-unit-tests.yml

```diff
@@ -0,0 +1,84 @@
+name: Run unit tests
+
+on:
+  push:
+    branches: [develop, master]
+    paths-ignore:
+      - './github/**'
+      - 'docs/**'
+      - 'test/e2e/**'
+      - '**/*.md'
+      - '**/*.csv'
+  pull_request:
+    types: [opened,reopened,synchronize]
+    paths-ignore:
+      - './github/**'
+      - 'docs/**'
+      - 'test/e2e/**'
+      - '**/*.md'
+      - '**/*.csv'
+
+jobs:
+  test-unit:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        shard: [1, 2, 3, 4, 5, 6]
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup environment
+        uses: metamask/github-tools/.github/actions/setup-environment@main
+
+      - name: test:unit:coverage
+        run: yarn test:unit:coverage --shard=${{ matrix.shard }}/${{ strategy.job-total }}
+
+      - name: Rename coverage to shard coverage
+        run: mv coverage/coverage-final.json coverage/coverage-${{matrix.shard}}.json
+
+      - uses: actions/upload-artifact@v4
+        with:
+          name: coverage-${{matrix.shard}}
+          path: coverage/coverage-${{matrix.shard}}.json
+
+  test-webpack:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup environment
+        uses: metamask/github-tools/.github/actions/setup-environment@main
+
+      - name: test:unit:webpack:coverage
+        run: yarn test:unit:webpack:coverage
+
+      - name: Rename coverage
+        run: mv coverage/coverage-final.json coverage/coverage-webpack.json
+
+      - uses: actions/upload-artifact@v4
+        with:
+          name: coverage-webpack
+          path: coverage/coverage-webpack.json
+
+  report-coverage:
+    runs-on: ubuntu-latest
+    needs:
+      - test-unit
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Download coverage from shards
+        uses: actions/download-artifact@v4
+        with:
+          path: coverage
+          pattern: coverage-*
+          merge-multiple: true
+
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@e28ff129e5465c2c0dcc6f003fc735cb6ae0c673
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
+          fail_ci_if_error: true
```

Comment:
seems like this file is renamed and I have missed it while updating the branch. I'll fix it